package.json

```diff
@@ -2,10 +2,28 @@
   "name": "redux-compose-reducer",
   "version": "1.2.0",
   "description": "Reducer's composer for Redux",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "unpkg": "dist/redux.js",
```

Comment:
Looks like copy / paste mistake. Should it be `dist/index.js`?

Reply:
https://github.com/kimrgrey/redux-compose-reducer/pull/3/commits/bd626cdafe82fede73f1ac648a1a9b6615166efd